api/src/pcapi/core/offerers/api.py

```diff
@@ -122,6 +123,30 @@ def update_venue(
             )
             upsert_venue_opening_hours(venue, opening_hours_data)
 
+    if external_accessibility_url is not offerers_constants.UNCHANGED:
+        external_accessibility_id = external_accessibility_url.split("/")[-2] if external_accessibility_url else None
+        external_accessibility_infos = {
+            "externalAccessibilityId": external_accessibility_id,
+            "externalAccessibilityUrl": external_accessibility_url,
+        }
+        target = (
+            venue.accessibilityProvider
+            if venue.accessibilityProvider is not None
+            else offerers_models.AccessibilityProvider()
+        )
```

Comment:
Je pense que tu peux l'écrire plus simplement : 
```suggestion
        target = venue.accessibilityProvider or offerers_models.AccessibilityProvider()
```
voire même directement en-dessous, ce qui aide à comprendre qu'on ne crée pas l'objet pour le stocker : 
```
        venue_snapshot.trace_update(
            external_accessibility_infos,
            target=venue.accessibilityProvider or offerers_models.AccessibilityProvider(),
            field_name_template="accessibilityProvider.{}",
        )
```